setup.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+setup.py
+A module that installs agol-validator as a module
+"""
+from glob import glob
+from os.path import basename, splitext
+
+from setuptools import find_packages, setup
+
+setup(
+    name='agol-validator',
+    version='0.9.0',
```

Comment:
**nitpick**: we start our package versions at 1.0.0

Reply:
With development status at `Beta`, I figured a sub 1.0 was more appropriate. I chose these both because with this refactor its definitely not ready yet. I plan on changing this to 1.0.0 for the PR where it fully integrates into forklift. Is there a different preferred way to handle that?

Reply:
yes, start at 1.0.0 no matter the development status. If you want to differentiate use 1.0.0-beta.1

Reply:
https://nodesource.com/blog/semver-tilde-and-caret/ This was the article that convinced scott and I to start at 1.0.0 back in 2014.